### PR TITLE
Fix JsonModel to build for Xcode 11.5

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Note: I am not updating travis at this time, just the swift package.

### CODE REVIEW

File: Research/Research/AssessmentTaskObject.swift [hunk 1]
```diff
@@ -62,6 +62,10 @@ extension RSDTaskType : DocumentableStringLiteral {
 }
 
 public final class TaskSerializer : AbstractPolymorphicSerializer, PolymorphicSerializer {
+    public var documentDescription: String? {
+        "`RSDTask` is the interface for running a task. It includes information about how to calculate progress, validation, and the order of display for the steps."
```
Comment (Erin-Mounts) [inline Research/Research/AssessmentTaskObject.swift]: Should this be split across multiple lines like the others? or does it even matter

File: Research/Research/RSDAsyncActionType.swift [hunk 2]
```diff
@@ -35,6 +35,19 @@ import Foundation
 import JsonModel
 
 public final class AsyncActionConfigurationSerializer : AbstractPolymorphicSerializer, PolymorphicSerializer {
+    public var documentDescription: String? {
+        """
+        `RSDAsyncActionConfiguration` defines general configuration for an asynchronous background
```
Comment (Erin-Mounts) [inline Research/Research/RSDAsyncActionType.swift]:
```suggestion
        `RSDAsyncActionConfiguration` defines general configuration for an asynchronous
```
(redundant with end of the sentence on next line)

File: Research/Research/RSDColorMappingThemeElementObject.swift [hunk 3]
```diff
@@ -67,6 +67,10 @@ extension RSDColorMappingThemeElementType : DocumentableStringLiteral {
 }
 
 public final class ColorMappingSerializer : AbstractPolymorphicSerializer, PolymorphicSerializer {
+    public var documentDescription: String? {
+        "`RSDColorMappingThemeElement` defines the colors to use on a given screen. Typically, this includes the background color for the header or a background color that is applied to the full screen."
```
Comment (Erin-Mounts) [inline Research/Research/RSDColorMappingThemeElementObject.swift]: multiple lines? ¯\\\_(ツ)\_/¯

File: Research/Research/RSDUIActionObject.swift [hunk 4]
```diff
@@ -81,6 +81,10 @@ extension RSDUIActionObjectType : DocumentableStringLiteral {
 }
 
 public final class ButtonActionSerializer : AbstractPolymorphicSerializer, PolymorphicSerializer {
+    public var documentDescription: String? {
+        "The `RSDUIAction` protocol can be used to customize the title and image displayed for a given action of the UI."
```
Comment (Erin-Mounts) [inline Research/Research/RSDUIActionObject.swift]: multiple lines? ¯\\\_(ツ)\_/¯

**Review by Erin-Mounts — APPROVED** [review]
A question (applicable in a few places) and a nit, otherwise looks good

File: Research/Research/AssessmentTaskObject.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply Research/Research/AssessmentTaskObject.swift]: ¯\_(ツ)_/¯ 

It's a good point and I should aim for consistency. ;)

**Pull request merged**